src/main/frontend/modules/shortcut/data_helper.cljs

```diff
@@ -190,7 +190,9 @@
 
 (defn shortcuts->commands [handler-id]
   (let [m (get @config/config handler-id)]
+    ;; NOTE: remove nil vals, since some commands are conditional
     (->> m
+         (filter (comp some? val))
```

Comment:
Would be great if we removed conditionals from commands altogether. They also caused issues when writing up the bb lang tasks. I'll see if I can address that soon